### DISCUSSION TITLE
Some wrong when reappling. "There are outstanding issues with your GitHub account. Please contact GitHub Support for more information. " shows up"

### DISCUSSION THREAD

**Discussion by BAp3r** [community/community#66586] [GitHub Education]
### Hi everyone,

There are outstanding issues with your GitHub account. Please contact GitHub Support for more information. Why it shows up. I chose to delay my graduation for personal reasons, so I am still a student at school. I've uploaded the certification by Ministry of Education of China and the snapshot of education e-mail page in the ticket, but the support team only responded once。
![image](https://github.com/community/community/assets/87560912/af242be8-424d-4e43-8c3b-510210e647a6)

**Comment by HorselessName** [comment]
Same here, I am seeing the same message in my account. No idea of what to do tho.
